src/features/game/events/landExpansion/startPotion.ts

```diff
@@ -16,9 +18,12 @@ type Options = {
 
 export const GAME_FEE = 320;
 
-export function startPotion({ state }: Options): GameState {
+export function startPotion({ state, action }: Options): GameState {
   return produce(state, (stateCopy) => {
     const { bumpkin, coins } = stateCopy;
```

Comment:
I would consider checking that the multiplier is a valid number (e.g. 1, 10, 50). Throw an error if not supported.

This could avoid someone providing a negative number or a really high number (e.g. 100000)